spec/pluralrules.html

```diff
@@ -347,7 +385,38 @@ <h1>ResolvePlural ( _pluralRules_, _n_ )</h1>
         1. Let _res_ be ! FormatNumericToString(_pluralRules_, _n_).
         1. Let _s_ be _res_.[[FormattedString]].
         1. Let _operands_ be ! GetOperands(_s_).
-        1. Return ! PluralRuleSelect(_locale_, _type_, _n_, _operands_).
+        1. Let _p_ be ! PluralRuleSelect(_locale_, _type_, _n_, _operands_).
+        1. Return the Record { [[PluralCategory]]: _p_, [[FormattedString]]: _s_ }.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-pluralruleselectrange" aoid="PluralRuleSelectRange">
+      <h1>PluralRuleSelectRange ( _locale_, _type_, _xp_, _yp_ )</h1>
+
+      <p>
+        When the PluralRuleSelectRange abstract operation is called with four arguments, it performs an implementation-dependent algorithm to map the plural form String values _xp_ and _yp_, representing the plural forms of the start and end of a range, to a resolved String value for the plural form of the range as a whole denoted by _type_ for the corresponding _locale_, or the String value *"other"*.
```

Comment:
```suggestion
        When the PluralRuleSelectRange abstract operation is called with four arguments, it performs an implementation-dependent algorithm to map the <emu-xref href="#sec-pluralruleselect">plural category</emu-xref> String values _xp_ and _yp_, respectively characterizing the start and end of a range, to a resolved String value for the plural form of the range as a whole denoted by _type_ (which must be *"cardinal"* or *"ordinal"*) for the corresponding _locale_, or the String value *"other"*.
```

But I think it would make more sense to name the { [[PluralCategory]], [[FormattedString]] } Record type and pass instances of _that_ to PluralRuleSelectRange (rather than just the [[PluralCategory]] field).